layouts/_default/list.html

```diff
@@ -15,16 +15,28 @@
       <ul class="pager">
         {{ if .Paginator.HasPrev }}
           <li class="previous">
+            {{ if $.Site.Params.uglyurls }}
+            <a href="{{ replaceRE "\\.html$" "/" .Permalink }}page/{{ .Paginator.Prev.PageNumber }}.html"
```

Comment:
📝 If you don't use replaceRE, URL will be broken. It looks: `exmaple.com/tags/tagname.htmlpage/2.html`

Reply:
Great infomation. Thanks!